docs/site.json

```diff
@@ -29,6 +29,7 @@
     }
   ],
   "headingIndexingLevel": 6,
+  "bootswatchTheme": "journal",
```

Comment:
This is in a temporary commit, to illustrate what modifying the theme looks like from the user's perspective.

Reply:
`"theme": "bootswatch-journal"` is more future proof, in case we add support for other themes?

Reply:
Nice, yeah that sounds better 👍

Reply:
@damithc - Updated! How does this look now?

Reply:
Looks good 👍